### ISSUE TITLE
API tvsearch by season only ever returns one entry

### ISSUE THREAD

**Issue by gkoh** [Murodese/pynab#186]
Currently running 1.6.0.
I recently tried doing a backlog search and was surprised to see only a single entry showing up.

The relevant api search query looks like this:
```
http://yourhost.com/api?apikey=<apikey>&limit=100&t=tvsearch&season=05&rid=<tvshow id>&maxage=500&cat=5030%2C5040
```

If I remove the season filter 'season=05' it returns many results in season 5 as expected but including other seasons (also as expected).
A manual check of the DB also confirms there are many more entries, they just don't seem to make it to newznab API output.
This seems to occur with any TV show I attempt to run a backlog search against.

Let me know if you need more details, I suspect the query into the DB is going wrong somewhere.

**Comment by brookesy2** [comment]
Does it work without max age? I haven't actually looked at the age calculation, but just a guess!

**Comment by Murodese** [comment]
I think this is going to be some weird sqlalchemy obscurity. Give me a bit to look.

**Comment by gkoh** [comment]
I did not actually try that!
I found removing the season filter was mostly right, so suspect
something there.

**Comment by gkoh** [comment]
Just merged bc03a98 into my sandbox, seems to work fine!
Thanks for that!
